packages/babel-preset-expo/src/index.ts

```diff
@@ -98,6 +98,21 @@ function babelPresetExpo(api: ConfigAPI, options: BabelPresetExpoOptions = {}):
     extraPlugins.push(require('@babel/plugin-transform-parameters'));
   }
 
+  if (!isDev && hasModule('metro-transform-plugins')) {
+    // Metro applies this plugin too but it does it after the imports have been transformed which breaks
+    // the plugin. Here, we'll apply it before the commonjs transform, in production, to ensure `Platform.OS`
+    // is replaced with a string literal and `__DEV__` is converted to a boolean.
+    // Applying early also means that web can be transformed before the `react-native-web` transform mutates the import.
+    extraPlugins.push([
+      require('metro-transform-plugins/src/inline-plugin.js'),
```

Comment:
Seems better to use the package's interface and load this with `require('metro-transform-plugins').inlinePlugin()`

Reply:
Pulls in more JavaScript that we aren't using, this import also recently changed from being in fbjs or something.